package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rascal",
-  "version": "7.0.0",
+  "version": "7.0.1",
```

Comment:
I'll bump this using `npm version [patch|minor|major]` so I get a tag. Not sure whether this is a patch though, since some of the deps have major version bumps. Will need to investigate a bit more closely.

Reply:
As there was no major changes to features or new feature additions, I thought of bumping up only the patch part. Using an automated version bumper sound better too.

Reply:
One of the bumps caused nodee v6 tests to fail (due to use of async). I've removed support for this but will bump to rascal 8.0.0

Reply:
Right. I did observe this from circle.ci logs. I guess, it should be fine to remove Node 6.x support at this time :-).